modules/payments/backend/modules/payments/models.py

```diff
@@ -20,6 +24,24 @@ class StripeUserProfile(models.Model):
 
 def create_stripe_profile(sender, instance, created, **kwargs):
     if created:
-        StripeUserProfile.objects.create(user=instance)
+        stripe_user = StripeService.create_user(instance)
+        if stripe_user:
+            StripeUserProfile.objects.create(user=instance, stripe_cus_id=stripe_user.id)
 
 post_save.connect(create_stripe_profile, sender=settings.AUTH_USER_MODEL, dispatch_uid="create_user_profile")
+
+
+class AppleIAPProduct(models.Model):
+    name = models.CharField(max_length=128, null=True, blank=True)
+    product_id = models.CharField(max_length=128)
+    is_active = models.BooleanField(default=False)
+
+    def __str__(self) -> str:
+        return "{}-{}".format(self.pk, self.name)
```

Comment:
```suggestion
        return f"{self.pk}-{self.name}"
```

Make use of f-string

Reply:
Bump